### DISCUSSION TITLE
Counter Display for relations

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17162] [Feature Requests]
Would be nice to have a counter `display` for the related list items in the table view of the collection list.

Like

| Status | Title | Images (Count)
|---|---|---|
| • | Foo | 2
| · | Bar | 9
| • | Boo | 0

**Comment by paescuj** [comment]
Hey @JonathanDoelan 👋 Seeing that you just started several new discussions, it seems like you have a great interest in Directus! 🚀  I'm glad to see that 😃 

Just a small side note:
Please make sure to search for similar discussions, before opening new ones. If there's already an existing discussion, it will have a much greater effect if you up-vote that one instead of starting a new one. This makes it easier for the team to see how much interest there is in a specific request.
In this case here, I think it matches with #13194 (and probably also #15431).
Moreover, it's always interesting to hear what the motivation behind an idea is (for example, by providing a use case).

**Reply by ghost** [reply]
@paescuj 

Yes, indeed, I have great interest in contributing to directus (read below). I also totally agree with you on searching for similar discussions before starting a new one. In this case I did not find the results that you have found. Seems like I have used the wrong key words. I do my best to prevent this from happening again.

---

About ten years ago I stopped working on my own CMS, because I simply did not find the time to maintain it. Since then I was constantly searching for an ideal solution, that corresponds to my vision. Hard to find. The path led me through CraftCMS, Backpack for Laravel, Laravel Nova over to Sanity. And I loved Sanity but it is [not open source](https://github.com/sanity-io/sanity/issues/3258). Then I came up with directus …

🎉🐰🎉

Never got things set up so quickly. I love so much about directus and I would love to share my thoughts on it with the community and help to improve the project.

That's why I wrote down everything I noticed while working with Directus. I have to admit that there is still a lot on my list 🙀 and I’ve already spend a lot of time to formulate them into discussions/issues/questions and, of course, to check if they exist. 

I hope that‘s OK for all of you!

**Reply by paescuj** [reply]
> [...] I also totally agree with you on searching for similar discussions before starting a new one. In this case I did not find the results that you have found. Seems like I have used the wrong key words. I do my best to prevent this from happening again.

Great! Admittedly, it's really difficult to find the right keywords sometimes! Thanks! It was also not meant as an accusation in any way, I just tend to mention it one time too much than too little 😄 

> About ten years ago I stopped working on my own CMS, because I simply did not find the time to maintain it. Since then I was constantly searching for an ideal solution, that corresponds to my vision. Hard to find. The path led me through CraftCMS, Backpack for Laravel, Laravel Nova over to Sanity. And I loved Sanity but it is [not open source](https://github.com/sanity-io/sanity/issues/3258). Then I came up with directus …
> 
> 🎉🐰🎉
> 
> Never got things set up so quickly. I love so much about directus and I would love to share my thoughts on it with the community and help to improve the project.

Wow, awesome! 🎉 I'm very glad to hear that! In fact, I feel very much the same way 🐰

> That's why I wrote down everything I noticed while working with Directus. I have to admit that there is still a lot on my list 🙀 and I’ve already spend a lot of time to formulate them into discussions/issues/questions and, of course, to check if they exist.
>
> I hope that‘s OK for all of you!

Of course, in my opinion it's always very exciting and valuable to hear about thoughts, new ideas and use cases!
In this sense, thank you for your contribution ❤️

**Reply by rijkvanzanten** [reply]
> Never got things set up so quickly. I love so much about directus and I would love to share my thoughts on it with the community and help to improve the project.
> 
> That's why I wrote down everything I noticed while working with Directus. I have to admit that there is still a lot on my list 🙀 and I’ve already spend a lot of time to formulate them into discussions/issues/questions and, of course, to check if they exist.
> 
> I hope that‘s OK for all of you!

Absolutely! Not gonna lie, sometimes a little overwhelming to see the large list of missed notifications when I wake up 👀 , but really appreciate your thoughts and great suggestions overall 😄

**Comment by ghost** [comment]
Alright then …

According to @paescuj’s comment, this is a duplicate of https://github.com/directus/directus/discussions/15431.
